typed-racket-doc/typed-racket/scribblings/reference/typed-contracts.scrbl

```diff
@@ -0,0 +1,156 @@
+#lang scribble/manual
+
```

Comment:
I read the docs - I don't really understand how the new `Con` and `FlatCon` types are used, when they appear, etc. They don't seem to be in any of the examples explicitly.